launcher/launcher/main.go

```diff
@@ -73,7 +76,7 @@ func main() {
 	defer serialConsole.Close()
 	logger.SetOutput(io.MultiWriter(os.Stdout, serialConsole))
 
-	logger.Println(welcomeMessage)
+	logger.Printf("%s, version: %s\n", welcomeMessage, Version)
```

Comment:
I think we sometimes key off the welcomeMessage. I'm not sure if it's a regex or pattern match, but, to be safe, move it to another line. 
```
logger.Println(welcomeMessage)
logger.Printf("Build commit: %s\n", Commit)
```

Reply:
done